### PR TITLE
refactor: replace bluebird .get

### PR DESCRIPTION
**Pull request by jedwards1211**
I may have missed one or two cases of `.get('stringLiteral')`.  ~~I haven't checked for `.get(identifier)`...~~ Okay, I didn't find any instances of `.get(identifier)` being in a bluebird promise chain

_Please make sure to review and check all of these items:_

- [ ] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [ ] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving?
- [ ] Have you added new tests to prevent regressions?
- [ ] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)?
- [ ] Did you update the typescript typings accordingly (if applicable)?
- [ ] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

File: test/integration/data-types.test.js [hunk 1]
```diff
@@ -469,9 +469,9 @@ describe(Support.getTestDialectTeaser('DataTypes'), () => {
       }).then(() => {
         return Model.findOne({ where: { id: 1 } });
       }).then(user => {
-        expect(user.get('float')).to.be.NaN;
-        expect(user.get('double')).to.eq(Infinity);
-        expect(user.get('real')).to.eq(-Infinity);
+        expect(user.then(obj => obj['float'])).to.be.NaN;
```
Comment (sushantdhiman) [inline test/integration/data-types.test.js:472]: `user` is not a promise

Comment (jedwards1211) [reply test/integration/data-types.test.js]: whoops

**Comment by sushantdhiman** [comment]
@jedwards1211 This needs rebase

**Comment by jedwards1211** [comment]
@sushantdhiman done

**Pull request merged**